pombola/nigeria/views.py

```diff
@@ -252,14 +155,7 @@ def get_area_from_pun(self, pun):
         forgiving in case of input error.
         """
 
-        while pun:
-            try:
-                area = Area.objects.get(codes__code=pun)
-                return area
-            except Area.DoesNotExist:
-                # strip off last component
-                pun = re.sub(r'[^:]+$', '', pun)
-                pun = re.sub(r':$', '', pun)
+        return requests.get('https://pu-lookup.herokuapp.com/lookup/{}'.format(pun)).json()
```

Comment:
Yep, definitely. This first commit is very much in the sling some code around to see if it works stage :)